docs/docusaurus/connections/connection.md

```diff
@@ -14,9 +14,9 @@ The connection protocol has two roles:
 
 ## Prerequisites
 
-1. Inviter and Invitee PRISM Agents up and running
+1. Inviter and Invitee Cloud Agents up and running
 
-## PRISM Agent endpoints overview
+## Identus Cloud Agent endpoints overview
```

Comment:
@yshyn-iohk , according to the [replacement table](https://docs.google.com/spreadsheets/d/1lR3abdg6qExXasUVqBlFouNu9FCB0I1ynMMZodQ6y8U/edit#gid=0), it shall be "Cloud Agent" instead of "Identus Cloudd Agent"

Reply:
@mkbreuningIOHK, let's collect the list of the changes and do these in the next PR